core/src/main/java/discord4j/core/event/domain/interaction/ChatInputInteractionEvent.java

```diff
@@ -29,44 +27,22 @@
 import java.util.Optional;
 
 /**
- * Dispatched when a user in a guild uses a Slash Command.
+ * Dispatched when a user uses a chat input command (formerly "slash command")
  * <p>
- * This event is dispatched by Discord.
+ * THis event is dispatched by Discord
```

Comment:
Don't know what you did to this line lol. Also, grammar; "Dispatched when _an_ user"

Reply:
"A user" is actually correct. English exceptions: <https://english.stackexchange.com/a/105117>
I will fix the stray capitals and missing period though.